packages/components/organisms/f-offers/src/components/Results.vue

```diff
@@ -0,0 +1,232 @@
+<template>
+    <div>
+        <content-cards
+            v-if="hasMounted"
+            :user-id="globalUserId"
+            :api-key="brazeApiKey"
+            :locale="$i18n.locale"
```

Comment:
Out of interest – do you need to pass through the locale? It should be globally available via `$i18n` I think.

Reply:
I think this is in place while we still supported sitecore, I will see if this is still needed